### PR TITLE
[servus] change to site new url and update api url

### LINKED ISSUE #29485: Servus TV not working anymore
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.06.06**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

.\youtube-dl.exe -v -F https://www.servustv.com/unterhaltung/v/aa-1vrgdg2yd1w12/
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'https://www.servustv.com/unterhaltung/v/aa-1vrgdg2yd1w12/']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[debug] exe versions: ffmpeg 2021-01-05-git-66deab3a26-full_build-www.gyan.dev, ffprobe 2021-01-05-git-66deab3a26-full_build-www.gyan.dev
[debug] Proxy map: {}
[generic] aa-1vrgdg2yd1w12: Requesting header
WARNING: Falling back on generic information extractor.
[generic] aa-1vrgdg2yd1w12: Downloading webpage
[generic] aa-1vrgdg2yd1w12: Extracting information
ERROR: Unsupported URL: https://www.servustv.com/unterhaltung/v/aa-1vrgdg2yd1w12/
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl31\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl31\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl31\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl31\build\youtube_dl\extractor\generic.py", line 3520, in _real_extract
youtube_dl.utils.UnsupportedError: Unsupported URL: https://www.servustv.com/unterhaltung/v/aa-1vrgdg2yd1w12/

## Description

Youtube-dl is not working with Servus TV, after Servus TV changed their website today.
Seems like every command is failing, even the example command for finding the available formats.
Example command: youtube-dl -v -F https://www.servustv.com/unterhaltung/v/aa-1vrgdg2yd1w12/

### LINKED ISSUE #29554: ServusTV no longer working
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dl version **2021.06.06**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
$ youtube-dl -v 'https://www.servustv.com/aktuelles/v/aa-28a3dbyxh1w11/'
[debug] System config: []
[debug] User config: [u'-f', u'bestvideo[ext=mp4]+bestaudio[ext=m4a]']
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'https://www.servustv.com/aktuelles/v/aa-28a3dbyxh1w11/']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.06.06
[debug] Python version 2.7.16 (CPython) - Linux-3.16.85+-aarch64-with-debian-10.0
[debug] exe versions: ffmpeg 4.1.6-1, ffprobe 4.1.6-1
[debug] Proxy map: {}
[generic] aa-28a3dbyxh1w11: Requesting header
WARNING: Falling back on generic information extractor.
[generic] aa-28a3dbyxh1w11: Downloading webpage
[generic] aa-28a3dbyxh1w11: Extracting information
ERROR: Unsupported URL: https://www.servustv.com/aktuelles/v/aa-28a3dbyxh1w11/
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/generic.py", line 2503, in _real_extract
    doc = compat_etree_fromstring(webpage.encode('utf-8'))
  File "/usr/local/bin/youtube-dl/youtube_dl/compat.py", line 2571, in compat_etree_fromstring
    doc = _XML(text, parser=etree.XMLParser(target=_TreeBuilder(element_factory=_element_factory)))
  File "/usr/local/bin/youtube-dl/youtube_dl/compat.py", line 2560, in _XML
    parser.feed(text)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1659, in feed
    self._raiseerror(v)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1523, in _raiseerror
    raise err
ParseError: syntax error: line 1, column 0
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/generic.py", line 3520, in _real_extract
    raise UnsupportedError(url)
UnsupportedError: Unsupported URL: https://www.servustv.com/aktuelles/v/aa-28a3dbyxh1w11/
```

## Description

ServusTV Mediathek no longer working. Looks like the site changed in some way.

### LINKED ISSUE #29825: ServusTV
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dl version **2021.06.06**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
PASTE VERBOSE LOG HERE
[debug] System config: []
[debug] User config: ['--no-check-certificate', '--geo-bypass-country', 'DE', '-
-hls-prefer-native', '--ffmpeg-location', 'C:\\Program Files (x86)\\Multimedia\\
FFMPEG\\bin']
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.servustv.com/unterhaltung/v/aa-1v
hrpqy1w2112/']
[debug] Encodings: locale cp1252, fs mbcs, out cp1252, pref cp1252
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.4.4 (CPython) - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-93439-gb073fb9eea, ffprobe N-93439-gb073fb9eea
[debug] Proxy map: {}
WARNING: Falling back on generic information extractor.
ERROR: Unsupported URL: https://www.servustv.com/unterhaltung/v/aa-1vhrpqy1w2112
/
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl
31\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl
31\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl
31\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl
31\build\youtube_dl\extractor\generic.py", line 3520, in _real_extract
youtube_dl.utils.UnsupportedError: Unsupported URL: https://www.servustv.com/unt
erhaltung/v/aa-1vhrpqy1w2112/


## Description

<!--
Provide an explanation of your issue in an arbitrary form. Provide any additional information, suggested solution and as much context and examples as possible.
If work on your issue requires account credentials please provide them or explain how one can obtain them.
-->

WRITE DESCRIPTION HERE
Hello!

seems that something has changed at servustv.com
In the past, youtube-dl worked perfect with this site, now I am getting the message "unsupported URL". 

command line:
youtube-dl.exe https://www.servustv.com/unterhaltung/v/aa-1vhrpqy1w2112/

The issue applies for any content on servustv.com. All other functions of youtube-dl such as -F etc are also affected.

### PR DESCRIPTION
**Pull request by kikuyan**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Read [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site)
- [x] Read [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) and adjusted the code to meet them
- [x] Covered the code with tests (note that PRs without tests will be REJECTED)
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Resolves #29485, resolves #29554, resolves #29825

Change to ServusTV new url (first commit) and update api url (second commit).

Note:
If anyone tries this PR, be sure to apply all the changes. Otherwise you won't get the expected results.

### CODE REVIEW

**Comment by embeddedc** [comment]
Will/Can this be merged?

**Comment by ChrLipp** [comment]
Tried to test your PR with https://www.servustv.com/volkskultur/v/aawg5x4ywdc94xyjv9ly/, got
`ERROR: Unsupported URL: https://www.servustv.com/volkskultur/v/aawg5x4ywdc94xyjv9ly/`

File: youtube_dl/extractor/servus.py [hunk 1]
```diff
@@ -18,30 +18,35 @@ class ServusIE(InfoExtractor):
                         (?:www\.)?
                         (?:
                             servus\.com/(?:(?:at|de)/p/[^/]+|tv/videos)|
-                            (?:servustv|pm-wissen)\.com/videos
+                            (?:servustv|pm-wissen)\.com/(?:videos|[^/]+/v)
                         )
                         /(?P<id>[aA]{2}-\w+|\d+-\d+)
```
Comment (crea-doo) [inline youtube_dl/extractor/servus.py:23]: When adding a additional `|\w+` for the id then also id's without a separator will work like the mentioned https://www.servustv.com/volkskultur/v/aawg5x4ywdc94xyjv9ly/

The full line should then be: `/(?P<id>[aA]{2}-\w+|\d+-\d+|\w+)`

**Comment by kikuyan** [comment]
Updated video-id regex.

File: youtube_dl/extractor/servus.py (hunk 1 above)
Comment (kikuyan) [reply youtube_dl/extractor/servus.py]: I was away and couldn't notice your review.

Comment (ChrLipp) [reply youtube_dl/extractor/servus.py]: Tested and works!

**Comment by shivan** [comment]
I tested with your branch, too, and works again. Great!

**Pull request closed without merging**